analysers/analyser_merge_police_FR.py

```diff
@@ -26,22 +26,31 @@
 class Analyser_Merge_Police_FR(Analyser_Merge):
     def __init__(self, config, logger = None):
         self.missing_official = {"item":"8190", "class": 1, "level": 3, "tag": ["merge"], "desc": T_(u"Police not integrated") }
+        self.possible_merge   = {"item":"8191", "class": 3, "level": 3, "tag": ["merge"], "desc": T_(u"Police, integration suggestion") }
+        self.update_official  = {"item":"8192", "class": 4, "level": 3, "tag": ["merge"], "desc": T_(u"Police update") }
+
         Analyser_Merge.__init__(self, config, logger,
-            "http://www.data.gouv.fr/fr/dataset/liste-des-points-d-accueil-de-la-gendarmerie-nationale-avec-geolocalisation",
-            u"Liste des points d'accueil de la gendarmerie nationale avec géolocalisation",
-            CSV(Source(attribution = u"data.gouv.fr:Ministère de l'Intérieur", millesime = "02/2016",
-                    fileUrl = "https://www.data.gouv.fr/s/resources/liste-des-points-d-accueil-de-la-gendarmerie-nationale-avec-geolocalisation/20160211-105304/ETALABexport_gn.csv"),
+            "https://www.data.gouv.fr/fr/datasets/liste-des-unites-de-gendarmerie-accueillant-du-public-comprenant-leur-geolocalisation-et-leurs-horaires-douverture/",
+            u"Liste des points d'accueil de la gendarmerie nationale",
+            CSV(Source(attribution = u"data.gouv.fr:Ministère de l'Intérieur", millesime = "10/2018",
+                    fileUrl = "https://www.data.gouv.fr/fr/datasets/r/d6a43ef2-d302-4456-90e9-ff2c47cac562"),
                 separator = ";"),
             Load("geocodage_x_GPS", "geocodage_y_GPS"),
             Mapping(
                 select = Select(
                     types = ["nodes", "ways"],
                     tags = {"amenity": "police"}),
-                conflationDistance = 1000,
+                conflationDistance = 500,
```

Comment:
It's required to lower the distance ?
There is existing Gendarmerie closer than 1000m ?

Reply:
the geoloc does not look that bad, and I would like to reduce the risk to conflate with police office that are not Gendarmerie.
Do you prefer I change it back to 1 km ?